termsaver/__init__.py

```diff
@@ -156,7 +174,8 @@ def getScreen():
         
         # Assign the important arguments on module init.
         verbose = True if args.verbose else False
-        if args.screen == None or (args.screen == None and args.help == True):
+        use_random = True if args.random else False
```

Comment:
the parser addition you made already takes care of this check with the "default" option... both the verbose and the rnadom should be just used as is from 'args'

Reply:
Yeah, I think this was done before that PR was merged, so I was trying to not use its functionality. Been a while since I've thought about it.